sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -958,6 +977,8 @@ public partial class HttpPipelineOptions
         public System.Collections.Generic.IList<Azure.Core.Pipeline.HttpPipelinePolicy> PerRetryPolicies { get { throw null; } }
         public Azure.Core.RequestFailedDetailsParser RequestFailedDetailsParser { get { throw null; } set { } }
         public Azure.Core.ResponseClassifier? ResponseClassifier { get { throw null; } set { } }
+        public System.Collections.Generic.IList<Azure.Core.RetryCondition> RetryConditions { get { throw null; } }
```

Comment:
- Library author has same knobs for modifying/replacing retry policy as end-user
- Precedence rules are the same as for response classification
  - Per-invocation takes precedent over
  - Per-client specified by end-user over
  - Library author modifications
  - Default pipeline implementation